versionData.json

```diff
@@ -189,6 +189,13 @@
 			{"id": "og-bgm-se",           "version": "1.0.6"}
 		]
 	},
+	{
+		"id": "Matsuribayashi Ch.8/voice-only",
+		"files": [
+			{"id": "voices",              "version": "0.0.1"},
+			{"id": "script",              "version": "0.0.1"}
+		]
+	},
```

Comment:
I think you added this section by mistake? Can you delete this and force push to update the commit?